packaging/installer/functions.sh

```diff
@@ -42,30 +26,15 @@ setup_terminal() {
         if [ $(($(tput colors 2>/dev/null))) -ge 8 ]; then
             # Enable colors
             TPUT_RESET="$(tput sgr 0)"
-            TPUT_BLACK="$(tput setaf 0)"
```

Comment:
The same for these definitions as well. I think we should keep them for completeness